### DISCUSSION TITLE
Solar System Viewer

### DISCUSSION THREAD

**Discussion by dmonty2** [Stellarium/stellarium#2650] [Q&A] [answered]
Is there a way to change perspectives in the Solar System Viewer?
i.e. In Stellarium I search for Solar System Viewer then CTRL+G.  But I would like to be able to have the sun stationary and pivot and zoom to observe the earths orbit and tilt.

Similar to this model
[https://www.earthspacelab.com/app/earth-revolution/](https://www.earthspacelab.com/app/earth-revolution/)

Thanks

**Comment by gzotti** [comment, marked as answer]
Use Alt+cursor keys and Alt+Home/End for distance. Admittedly though, the current state of attitude control is not satisfactory. Hopefully still later this year, but there are more urgent things to repair.
